mitmproxy/tools/main.py

```diff
@@ -76,6 +76,7 @@ def run(
         sys.exit(1)
 
     try:
+        # TODO: `--set` should take precedence over config.yaml, but we do this first for now as it may modify confdir.
```

Comment:
I've created an issue to track this and not forget https://github.com/mitmproxy/mitmproxy/issues/4047